### PR TITLE
Add 1.10.0 firmware

### PR DESCRIPTION
**Pull request by tsusanka**
I am intentionally doing this myself so I understand the process.

### CODE REVIEW

**Comment by tsusanka** [comment]
I should also modify https://github.com/trezor/webwallet-data/blob/master/firmware/1/releases.json right? And then copy paste to Connect instead of doing it [there](https://github.com/trezor/connect/pull/805) directly?

**Comment by prusnak** [comment]
> I should also modify https://github.com/trezor/webwallet-data/blob/master/firmware/1/releases.json right?

Right.

> And then copy paste to Connect instead of doing it 

Szymon usually takes care of updating this in Connect, but I guess you can send PR to Connect too (but only after this PR has been merged).

File: firmware/1/releases.json [hunk 1]
```diff
@@ -1,4 +1,18 @@
...
+		"required": false,
+		"version": [1, 10, 0],
+		"bootloader_version": [1, 10, 0],
+		"min_bridge_version": [2, 0, 25],
+		"min_firmware_version": [1, 6, 2],
+		"min_bootloader_version": [1, 5, 0],
+		"url": "data/firmware/1/trezor-1.10.0.bin",
+		"url_bitcoinonly": "data/firmware/1/trezor-1.10.0-bitcoinonly.bin",
+		"fingerprint": "595ba3e8e887cba185e03098f9538e18164f72f9fc82445e691abcd03e5cf0a4",
+		"fingerprint_bitcoinonly": "20a4068c34ff6dd7d8c510350409376cf7ea744ba668fdcf16da8f1d81fed289",
```
Comment (tsusanka) [inline firmware/1/releases.json:12]: I am not 100% I have done this correctly. I have used trezorctl firmware-update -f to find it out which is kind of lame. Please doube-check.

File: firmware/1/releases.json [hunk 2]
```diff
@@ -1,4 +1,18 @@
 [
+	{
+		"required": false,
+		"version": [1, 10, 0],
+		"bootloader_version": [1, 10, 0],
+		"min_bridge_version": [2, 0, 25],
+		"min_firmware_version": [1, 6, 2],
+		"min_bootloader_version": [1, 5, 0],
```
Comment (tsusanka) [inline firmware/1/releases.json:8]: I have copied & paste it from the latest release but is this correct? You need to install the intermediary FW first nope?

**Comment by tsusanka** [comment]
Added intermediary FW. I do not touch releases.json for that one right?

**Comment by tsusanka** [comment]
Removed Intermediary FW due to https://github.com/trezor/trezor-firmware/issues/1597.

File: firmware/1/releases.json (hunk 1 above)
Comment (prusnak) [reply firmware/1/releases.json:12]: Correct!

File: firmware/1/releases.json (hunk 2 above)
Comment (prusnak) [reply firmware/1/releases.json:8]: This is correct for now. 1.10.0 still can be installed without the interim FW.

**Pull request merged**